src/renderers/dom/shared/ReactDOMDebugTool.js

```diff
@@ -40,9 +41,11 @@ function emitEvent(handlerFunctionName, arg1, arg2, arg3, arg4, arg5) {
 
 var ReactDOMDebugTool = {
   addDevtool(devtool) {
+    ReactDebugTool.addDevtool(devtool);
```

Comment:
>maybe it makes more sense to only expose each platform-specific one

Let’s go with this assumption for now. In this case it makes sense to automatically setup forwarding.